Blog post on Securing API with Mulesoft

mraible opened the pull request:
NOTE: There's some unresolved image issues marked with "todo".

aaronpk commented on _source/_posts/2018-07-06-secure-api-mulesoft.md:
````diff
@@ -0,0 +1,452 @@
+Now let's configure your environment. The Okta API Center uses the [dotenv npm package](https://www.npmjs.com/package/dotenv) to help manage environment variables, so you can add these values to a `.env` file if you wish. There is a `.env_example` file included in the repo for reference, which looks like this:
+
+```bash
+OKTA_TENANT="https://dev-292102.oktapreview.com"
````
use `{yourOktaOrg}` or whatever that placeholder text is here instead.

aaronpk commented on _source/_posts/2018-07-06-secure-api-mulesoft.md:
```diff
@@ -0,0 +1,452 @@
+
+To set up your Okta tenant with all of the components needed for this example such as users, groups, authorization servers, etc., we'll run a couple bootstrap scripts.
+
+**NOTE:** if you'd like to set up your tenant manually, rather than using the bootstrap tool, there are step-by-step instructions at the end of this article.
```
Add hyperlink to a fragment ID to those instructions.

mraible commented:
> Looks like the last image is missing: okta_add_rule.png

Yes, there was a couple of images missing. I've fixed those.

> Did we completely abandon the style guide of bolding button text for this?

> Everything past "Instructions for those not using the bootstrap tool:" looks like it wasn't yet formatted. It has a bunch of leading whitespace that looks like it's intended to show indentation, but just gets collapsed. The numbered list items have no spacing above them.

Brent told me not to spend any more time with editing this post, so I didn't convert "click here" to **click here**. Also, the ending instructions weren't formatted in Markdown in the Google Docs, so I converted it to Markdown, but didn't do any further editing.
